src/test/scala/is/hail/PipelineSuite.scala

```diff
@@ -23,7 +23,10 @@ class PipelineSuite extends SparkSuite {
     val pedigree = Pedigree.read("src/test/resources/sample.fam", hadoopConf)
     qc.mendelErrors(pedigree)._1.export(mendelBase)
     qc.count()
-    qc.filterVariantsExpr("va.qc.AF > 0.01 && va.qc.AF < 0.99")
-      .pca("sa.scores")
```

Comment:
Let's just delete PipelineSuite.  It's one of the first things we wrote and I think everything either has its own tests or is tested better e.g. in the docs and tutorials.

Reply:
💥